symphony-bdk-template/symphony-bdk-template-api/src/main/java/com/symphony/bdk/template/api/TemplateException.java

```diff
@@ -10,7 +10,7 @@
  * if template is malformed or if some parameters are missing when calling {@link Template#process(Object)}
  */
 @API(status = API.Status.STABLE)
-public class TemplateException extends Exception {
+public class TemplateException extends RuntimeException {
```

Comment:
We might to update documentation, please take a look

Reply:
I didn't find any existing documentation mentioning the `TemplateException`